### PR TITLE
Add a first local kubernetes deployment

### PR DESCRIPTION
**Pull request by Hustenbonbon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
 
+"oTTo (openspace Time Table organizer) is an open source service for managing open spaces such as barcamps in which participants can add and move sessions during the event and the time table is adjusted  in real time. It uses Vue, Spring Boot with Kotlin and Postgresql. 
 
-Build
 
-`docker run -it --rm --name otto -v "$(pwd)":/usr/src/mymaven -w /usr/src/mymaven maven:3.9-eclipse-temurin-17 mvn clean install`
+# Local Setup 
+
+
+We use [direnv](https://direnv.net/) to manage the local environment. You can install it with your package manager or follow the instructions on the website.
```
Comment (kaidobit) [inline README.md:9]: direnv + nix package manager, not only direnv

**Comment by kaidobit** [comment]
when use helm id expect its whole config to be inside a values.yml, its seems unusual to split up config across multiple files (dev-db-config.yml + values.yml)
same for skaffold

**Comment by Hustenbonbon** [comment]
> when using helm i'd expect its whole config to be inside a values.yml, its seems unusual to split up config across multiple files (dev-db-config.yml + values.yml) same for skaffold
> 
> is there a reason to split those?

skaffold.yaml: The split makes it possible to depend the deploy of backend to postgres, so backend only deploy after postgres is up. this is currently (and for the foreseeable future) the only way to model that dependenc.

values.yaml: It's separate helm charts, the one of postgres is only meant for dev setup, the one for backend is going to be for people who want to deploy it in their cluster

**Comment by kaidobit** [comment]
i agree with this beeing currently a valid workaround, however a better approach, would be creating a init-container in the deployment which wait for postgres to be ready and then shuts down

this would trigger the main-container of your deployment to startup

here is an example: https://medium.com/@xcoulon/initializing-containers-in-order-with-kubernetes-18173b9cc222

**Pull request merged**